sales_team_multicompany/security/ir.rule.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,groups:id,perm_read,perm_write,perm_create,perm_unlink,domain_force
+sales_team.crm_team_rule,crm.team multi-company,sales_team.model_crm_team,,True,False,False,False,"['|', ('company_ids', '=', 'user.company_id.id'), ('company_ids','child_of',[user.company_id.id])]"
```

Comment:
@pedrobaeza, Thanks for your remarks, I corrected.

Le 14/03/2016 15:39, Pedro M. Baeza a écrit :
>
> In sales_team_multicompany/security/ir.rule.csv 
> <https://github.com/OCA/multi-company/pull/22#discussion_r56010150>:
>
> > @@ -0,0 +1,2 @@
> > +id,name,model_id:id,groups:id,perm_read,perm_write,perm_create,perm_unlink,domain_force
> > +sales_team.crm_team_rule,crm.team multi-company,sales_team.model_crm_team,,True,False,False,False,"['|', ('company_ids', '=', 'user.company_id.id'), ('company_ids','child_of',[user.company_id.id])]"
>
> The rules are usually put in XML and with a noupdate="1" attribute for 
> allowing to change the rule manually.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/multi-company/pull/22/files#r56010150>.
>